ui-tests/pom.xml

```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <webapp.url>http://devtest01.openmrs.org:8080/openmrs</webapp.url>
+        <webapp.url>http://int02.openmrs.org:8080/openmrs</webapp.url>
```

Comment:
Actually please use int-refapp, which is a clearer alias for our purposes. And we don't need the :8080 anymore.